### DISCUSSION TITLE
Disable form cancel popup

### DISCUSSION THREAD

**Discussion by c-gross** [orchidsoftware/platform#2644] [Q&A] [answered]
Since updating to Orchid 14.2.1 I have the problem with the new form cancel popup "Do you really want to leave? You have unsaved changes." when leaving or refreshing a page with unsaved changes.

On a very customized page inside the admin panel it's causing big problems for us. I need to disable this new feature somehow global or for specific pages.

It seems to be located in: vendor/orchid/platform/resources/js/controllers/form_controller.js
I have no clue how I could change/overwrite that controller.js or disable this check in any other way.

Any solution that works is welcome. I really need to get this fixed!

**Comment by tabuna** [comment, marked as answer]
You can disable this behavior in the configuration file:

https://github.com/orchidsoftware/platform/blob/262e3c80cafbae673c5b16b754211c2b5cf54250/config/platform.php#L295-L305

Or set this behavior in a particular screen. For example, enable/disable

https://github.com/orchidsoftware/platform/blob/262e3c80cafbae673c5b16b754211c2b5cf54250/src/Screen/Screen.php#L345-L351

**Comment by c-gross** [comment]
Thank you, it's working.
There is also the config for the workspace full or compact template I was search for some time ago.

I copied things like the user/role files after the upgrade but I never checked the config folder, since it's not getting updated after composer update.
